buffer/src/main/java/io/netty/buffer/PoolArena.java

```diff
@@ -31,7 +31,7 @@
 import static java.lang.Math.max;
 
 abstract class PoolArena<T> extends SizeClasses implements PoolArenaMetric {
-    static final boolean HAS_UNSAFE = PlatformDependent.hasUnsafe();
+    private static final boolean HAS_UNSAFE = PlatformDependent.hasUnsafe();
```

Comment:
why we were caching it at all?
@normanmaurer @chrisvest ?

Reply:
I don't know. There's no reason to AFAIK.